stubs/netifaces/netifaces.pyi

```diff
@@ -0,0 +1,74 @@
+from typing import Dict, List, Tuple, Union, Literal
+
+AF_12844: int
```

Comment:
These should all be marked `Final[int]`.

Reply:
Done as well.